src/extractors/custom/japan.zdnet.com/index.js

```diff
@@ -0,0 +1,31 @@
+export const JapanZdnetComExtractor = {
+  domain: 'japan.zdnet.com',
+
+  title: {
+    selectors: ['h1'],
+  },
+
+  author: {
+    selectors: ['.author'],
+  },
+
+  date_published: {
+    selectors: ['.author'],
```

Comment:
Let's use the `article:published_time` `meta` property instead. At this point, that `meta` selector feels a bit more solid, in case the website structure is changed and the author and date are no longer in the same `.author` element

Reply:
Thank you.
I fixed the date_published selector.